pubber/src/main/java/daq/pubber/Pubber.java

```diff
@@ -357,24 +362,32 @@ private void initializeDevice() {
       pullDeviceMessage();
     }
 
-    info(String.format("Starting pubber %s, serial %s, mac %s, extra %s, gateway %s",
+    info(String.format("Starting pubber %s, serial %s, mac %, gateway %s",
         configuration.deviceId, configuration.serialNo, configuration.macAddr,
-        configuration.extraField,
         configuration.gatewayId));
 
     deviceState.system.operational = true;
     deviceState.system.serial_no = configuration.serialNo;
     deviceState.system.hardware.make = "BOS";
     deviceState.system.hardware.model = "pubber";
     deviceState.system.software = new HashMap<>();
-    deviceState.system.software.put("firmware", "v1");
+    deviceState.system.software.put("firmware", "v1");    
     deviceState.system.last_config = new Date(0);
-    devicePoints.extraField = configuration.extraField;
 
-    if (configuration.extraPoint != null && !configuration.extraPoint.isEmpty()) {
-      addPoint(makePoint(configuration.extraPoint,
+    // Pubber runtime options
+    if (configuration.options.extraField) {
+      devicePoints.extraField = "extra_field";
+    }
+
+    if (configuration.options.extraPoint) {
+      addPoint(makePoint("extra_point",
```

Comment:
ditto -- extra_point here would be useful

Reply:
extra point value now taken from string